### DISCUSSION TITLE
Drop all internet traffic of sandboxed app and allow only localhost/127.0.0.1

### DISCUSSION THREAD

**Discussion by osevan** [netblue30/firejail#5065] [Q&A] [answered]
How I can drop egress traffic (outgoing traffic to internet) and allowing only localhost/127.0.0.1 traffic inside sandbox?

Thanks and

Best regards

**Comment by rusty-snake** [comment, marked as answer]
* #3439

**Reply by osevan** [reply]
> #3439

I read that now, but for clear understanding  I need example how i can realize that.

Especially with netfilter + net flag. ..


Thanks and

best regards

**Comment by osevan** [comment]
I need similar to sytemd ip hardening in firejail

 systemctl set-property mariadb.service IPAddressDeny=any IPAddressAllow=localhost

Like here

https://0pointer.net/blog/ip-accounting-and-access-lists-with-systemd.html

**Comment by osevan** [comment]
We need easy solution with onetime flag like:

no-internet-access

**Comment by osevan** [comment]
I will test tomorrow this one and report. 

firejail --net=eth0 --netfilter=/etc/firejail/tcpserver.net,3306 server-program

**Comment by osevan** [comment]
Not working .

I did netfilter with net enp0s9 because its my wired card inside VM plus  netfilter /etc/firejail/tcpserver.net,3306

Inside profile :
net enp0s9
netfilter /etc/firejail/tcpserver.net,3306

But now I cant access my mariadb on 127.0.0.1 anymore.
When I remove net everything works. 

And I can see  with ss -tualp mysql port. ..

So for me not working.

**Comment by osevan** [comment]
I did with brctl add br0 
bridge network 

And assigned with ifconfig br0 10.10.20.1/24 
IP address


My ip inside sandbox is 10.10.20.196

I can ping this address, but port 3306 cant accessable even without netfilter

**Comment by osevan** [comment]
Errorlog from mysql side 
!
![IMG_20220323_111622_edit_161743080365943](https://user-images.githubusercontent.com/77795961/159678871-602ccf90-6a81-4304-8bff-4781ca6b6288.jpg)

**Comment by osevan** [comment]
![IMG_20220323_111526_edit_161683627077931](https://user-images.githubusercontent.com/77795961/159676574-4c7156ed-457d-4c64-bcb1-a57096094589.jpg)

**Comment by osevan** [comment]
I solved the issue now.

Thanks for help
